src/commands/game_commands/play.ts

```diff
@@ -170,7 +176,12 @@ export default class PlayCommand implements BaseCommand {
                     sendErrorMessage(messageContext, { title: ignoringOldGameTypeTitle, description: oldGameTypeInstructions, thumbnailUrl: KmqImages.DEAD });
                 }
 
-                gameSession = new GameSession(textChannel.id, voiceChannel.id, textChannel.guild.id, gameOwner, GameType.CLASSIC);
+                if (isCompetitionMode && !COMPETITION_MODERATOR_IDS.includes(message.author.id)) {
```

Comment:
What if we just restrict the command to the debug channel, instead of checking for moderator id

Reply:
The debug channel will be different from the one that hosts the competition